libkbfs/reporter_kbpki.go

```diff
@@ -241,9 +241,6 @@ func (r *ReporterKBPKI) SuppressNotifications(
 // Shutdown implements the Reporter interface for ReporterKBPKI.
 func (r *ReporterKBPKI) Shutdown() {
 	r.canceler()
-	close(r.notifyBuffer)
-	close(r.notifySyncBuffer)
-	close(r.suppressCh)
```

Comment:
We still probably want these to panic so that we are made aware of races (instead of infinitely blocking).